.github/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# Changes from release 2018/09 to 2018/10
+3. AIRAC (1810) - Set the squawks allocated to EGBE_R_TWR to 0420
```

Comment:
Need to add "- thanks to @1adamf (Adam Farquharson)"

Also something like "Amended EGBE squawk code allocation" - don't need to describe position and the actual squawk